packages/third-parties/pulse/src/PulseModule.spec.ts

```
@@ -137,7 +139,46 @@
         expect(pulseModule.pulse.close).toHaveBeenCalledWith({force: true});
       });
     });
-  });
+    describe("_collection", () => {
+      it("should expose _collection", () => {
+        const pulseModule = PlatformTest.get<any>(PulseModule)!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)